### ISSUE TITLE
Failure in ProcessTest.Process_MainModule in CI build

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#606]
System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule [FAIL]
        Process_MainModule failed
        Stack Trace:
 c:\j\workspace\dotnet_corefx_prtest\Config\Debug\OS\Windows_NT\Tests\false\label\windows\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTest.cs(242,0): at System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule()
  Finished:    System.Diagnostics.Process.Tests

**Comment by stephentoub** [comment]
This failed again in a Windows debug CI run, though at a different line:

MESSAGE:
System.NullReferenceException : Object reference not set to an instance of an object.
+++++++++++++++++++
STACK TRACE:
at System.Diagnostics.ProcessModule.get_ModuleName() in d:\j\workspace\dotnet_corefx_windows_debug\src\System.Diagnostics.Process\src\System\Diagnostics\ProcessModule.cs:line 37 at System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule() in d:\j\workspace\dotnet_corefx_windows_debug\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTest.cs:line 225

**Comment by stephentoub** [comment]
Failed again:

 System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule [FAIL]
        System.NullReferenceException : Object reference not set to an instance of an object.
        Stack Trace:
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Diagnostics.Process\src\System\Diagnostics\ProcessModule.cs(37,0): at System.Diagnostics.ProcessModule.get_ModuleName()
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTest.cs(225,0): at System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule()

**Comment by stephentoub** [comment]
Another, from http://dotnet-ci.cloudapp.net/job/dotnet_corefx_windows_release_prtest/238/console
```
System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule [FAIL]
        System.NullReferenceException : Object reference not set to an instance of an object.
        Stack Trace:
           d:\j\workspace\dotnet_corefx_windows_release_prtest\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTest.cs(221,0): at System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule()
  Finished:    System.Diagnostics.Process.Tests
```

**Comment by stephentoub** [comment]
Another, from http://dotnet-ci.cloudapp.net/job/dotnet_corefx_windows_release_prtest/240/console
```
System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule [FAIL]
        System.NullReferenceException : Object reference not set to an instance of an object.
        Stack Trace:
d:\j\workspace\dotnet_corefx_windows_release_prtest\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTest.cs(221,0): at System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule()
  Finished:    System.Diagnostics.Process.Tests
```

**Comment by stephentoub** [comment]
Another, from http://dotnet-ci.cloudapp.net/job/dotnet_corefx_windows_release_prtest/430/console
```
System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule [FAIL]
        System.NullReferenceException : Object reference not set to an instance of an object.
        Stack Trace:
           d:\j\workspace\dotnet_corefx_windows_release_prtest\src\System.Diagnostics.Process\tests\System.Diagnostics.Process.Tests\ProcessTest.cs(219,0): at System.Diagnostics.ProcessTests.ProcessTest.Process_MainModule()
  Finished:    System.Diagnostics.Process.Tests
```

**Comment by pallavit** [comment]
I briefly looked at the code defect however any fix will give us a divergent behavior from the desktop. I don't think that should be a concern in this particular case but that means we might have to go through some process. 

I will send a PR to disable the test for now, and work on the fix in parallel.

**Comment by stephentoub** [comment]
Fixed by #1301
